### ISSUE TITLE
Not Able To Start

### ISSUE THREAD

**Issue by RFedz007989** [Herick-Asmani/AI-Photo-Editing-with-Inpainting#1]
Hi,

I cloned the repo & also installed the requirements, but unable to start.
Can you pls tell what is the command line to start.
Thank You

**Comment by Herick-Asmani** [comment]
@RFedz007989 did you try running the starter.ipynb first?

**Comment by RFedz007989** [comment]
> @RFedz007989 did you try running the starter.ipynb first?

No. Thanks I will do that & let you know if that works for me or not.

**Comment by RFedz007989** [comment]
As you said I tried, but unable to open this (starter.ipynb)

Pls check folder & way I installed & Pls let me know is this correct or is there something missing.
Thank You
![Not Able To Start](https://github.com/Herick-Asmani/AI-Photo-Editing-with-Inpainting/assets/158298313/77f81f70-69d8-4a2e-9587-15ac99718682)
